portfolio/src/main/webapp/mapsScript.js

```diff
@@ -50,15 +100,29 @@ function addlocations(map) {
                 position: {lat: comment.lat, lng: comment.lng},
                 map: map,
             });
-            marker.addListener('mouseover', function() {
-                infowindow.open(map, marker);
-            });
-            marker.addListener('mouseout', function() {
-                infowindow.close();
+            marker.addListener('click', function() {
+                if (i%2 === 0) {
```

Comment:
rather than keeping track is there a way to determine if the window is currently open?

Reply:
Alternatively, stick to allowing the user to close the window with the 'x' button.

Reply:
I checked the documentation but don't think there's a way to check if window is open

Reply:
Okay!